core/include/SharedPlanePtr.h

```diff
@@ -39,6 +39,21 @@ namespace genfit {
  */
     typedef std::shared_ptr< genfit::DetPlane > SharedPlanePtr;
 
+    /**
+     * Class allowing to create a SharedPlanePtr from a DetPlane.
```

Comment:
...from Python

Reply:
I have updated the comment to make that clear.